include/onnxruntime/core/graph/graph.h

```diff
@@ -18,6 +18,7 @@
 
 #if !defined(ORT_MINIMAL_BUILD)
 #include "onnx/defs/schema.h"
+#include "core/common/inlined_containers.h"
 #else
```

Comment:
I would have expected `#include "core/common/inlined_containers_fwd.h"` to be sufficient,with the .cc including this header.